os/drivers/input/touchscreen.c

```diff
@@ -144,13 +145,6 @@ static int touch_open(FAR struct file *filep)
 		}
 		priv->tp_buf.size = CONFIG_TOUCH_BUFSIZE;
 
-		if (priv->ops && priv->ops->touch_enable) {
```

Comment:
When touch is enabled when silent mode booting?
As I know, when knockknock event happens, it turns on LCD and enabled touch.
@ewoodev Is it right?